charts/k8s-infra/values.yaml

```diff
@@ -227,6 +222,8 @@ presets:
       # - azure
       # - heroku
       - system
+      - env
```

Comment:
In order of precedence because it is important here:
- env
- k8snode
- (cloud platform)
- system

For cloud platform, please read from global.cloud

This will also close https://github.com/SigNoz/charts/issues/513

Reply:
Updated